src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerSearchService.kt

```diff
@@ -105,6 +123,16 @@ class PrisonerSearchService(
   private fun getIndex(): String{
     return indexStatusService.getCurrentIndex().currentIndex.indexName
   }
+
+  fun findByListOfprisonerNumbers(prisonerListCriteria: PrisonerListCriteria): List<Prisoner> {
```

Comment:
Slight typo in the function name - should be camel case; the 'P' in prisoner should be uppercase